Gemfile

```diff
@@ -1,7 +1,7 @@
 source "https://rubygems.org"
-
-# This should move to the next release, see https://github.com/CocoaPods/Cork/pull/10
-gem "cork", git: "https://github.com/CocoaPods/Cork.git"
 gemspec
 
+# This should get removed when > 3.7.0 comes out
+gem "gitlab", git: "https://github.com/NARKOZ/gitlab.git", branch: "master"
```

Comment:
Yes, the official gem uses JSON 1.7 ( because it's stored as json in rubygems), which conflicts with _something_ which makes my testing tooling not work - so I made https://github.com/NARKOZ/gitlab/pull/243 and used something which would use the ruby gemspec, and not hit the same problems )